### ISSUE TITLE
Unable To Compile 'TextBox_themeresources.xaml'

### ISSUE THREAD

**Issue by MajidAbuRmelah** [microsoft/microsoft-ui-xaml#7685]
### Describe the bug

When I copy and paste the content of the file _/dev/CommonStyles/TextBox_themeresources.xaml_ into my resource dictionary file I get the following error:
`Type 'ThemeResource' used after '{' must be a Markup Extension. Error code 0x03e9.	'My_Project_Name'`

This error starts at the line 223 and continues to line 226 in that xaml file. This is the code that causes the issue:
```
<contract6Present:Setter Property="UseSystemFocusVisuals" Value="{ThemeResource IsApplicationFocusVisualKindReveal}" />
<contract7Present:Setter Property="ContextFlyout" Value="{StaticResource TextControlCommandBarContextFlyout}" />
<contract7Present:Setter Property="SelectionFlyout" Value="{StaticResource TextControlCommandBarSelectionFlyout}" />
<contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
```

### Steps to reproduce the bug

1. Create a new resource dictionary file.
2. Copy the code from /dev/CommonStyles/TextBox_themeresources.xaml.
3. Replace the code in step 1 with the code in step 2.

### Expected behavior

Be able to compile the app. The controls must look exactly as the control in the Win UI 3.

### NuGet package version

WinUI 3 - Windows App SDK 1.2 Experimental: 1.2.220727.1-experimental1

### Windows app type

- [x] UWP
- [ ] Win32

### Windows version

Windows 11 (22H2): Build 22621

**Comment by dpaulino** [comment]
I'm getting this as well. Anyone know why?

**Comment by aquinn39** [comment]
Did anyone ever find a fix for this? I get this as well.

**Comment by ghost1372** [comment]
> Did anyone ever find a fix for this? I get this as well.

it seems that fixed in v1.7
https://github.com/microsoft/microsoft-ui-xaml/issues/2556#issuecomment-2644369897

**Comment by aquinn39** [comment]
Hmm yeah looks like that's WinUI 3. Seems to still be broken in WinUI 2/UWP.
